### PR TITLE
Engine vacuum tool and setup plugin feature page

### PR DESCRIPTION
**Pull request by rgolangh**
### Motivation
This page describes the the engine-vacuum cli tool and the setup plugin that uses it
and is the actual result of the work on https://bugzilla.redhat.com/show_bug.cgi?id=1388430

### Fixes issue https://bugzilla.redhat.com/show_bug.cgi?id=1388430

Please review:
@mykaul @didib @mwperina @mgoldboi

### CODE REVIEW

File: source/develop/release-management/features/infra/Engine_vacuum.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Engine Vacuum
+
+# Description
+This page describes the **_engine-vacuum_** cli tool and the **_vacuum setup-plugin_** that uses it.
+
+# Motivation
+
+> "PostgreSQL databases require periodic maintenance known as vacuuming.
+For many installations, it is sufficient to let vacuuming be performed by the autovacuum daemon"
+> - routine-vacuummin, [postgres documentation][1]
```
Comment (mykaul) [inline source/develop/release-management/features/infra/Engine_vacuum.md]: routine-vacuummin -> missing 'g'.

File: source/develop/release-management/features/infra/Engine_vacuum.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+Apart from updating the tables and removing garbage from the tables,
+full vacuum is the only vacuum operation that can reclaim disk space back to the operating system.
+If the engine db is falling behind in collecting garbage,(autovacuum doesn't perfom optimally) we will
+see disk growth in time as a result.
```
Comment (mykaul) [inline source/develop/release-management/features/infra/Engine_vacuum.md]: disk usage growth

File: source/develop/release-management/features/infra/Engine_vacuum.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+## engine-vacuum
+  engine-vacuum [OPTION]...
+
+Located under `/bin/engine-vacuum`, invoking this can perform vacuum, vacuum full
+and analyze against the installed engine db, optionally specifying the tables,
+without needing to specifying the engine db, user and password.
+```text
+engine-vacuum --help
+
+  -a          - run analyze, update optimizer statistics
```
Comment (mykaul) [inline source/develop/release-management/features/infra/Engine_vacuum.md]: It's not very clear what 'optimizer' is - first time it's mentioned.

File: source/develop/release-management/features/infra/Engine_vacuum.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+- Fail install on error: _Yes_, output should go to the log
+
+## Security
+The credentials of the user, engine db and password
+are taken from /etc/ovirt-engine/engine.conf.d/10-setup-database and passed
+using .pgpass file thus not used as a command argument and not in risk to
+go into history, log or similar. See [pgpass documentation][2] for more details.
+
+## TODO
+- Try to give a ruff estimation of how long this is going to take. We can count
```
Comment (mykaul) [inline source/develop/release-management/features/infra/Engine_vacuum.md]: ruff -> rough

File: source/develop/release-management/features/infra/Engine_vacuum.md (hunk 3 above)
Comment (rgolangh) [inline source/develop/release-management/features/infra/Engine_vacuum.md]: added links inline (in the examples section) and added the relevant wiki page at the reference section

**Comment by rgolangh** [comment]
@doron-fediuck Header added

**Pull request merged**